src/main/java/io/github/moulberry/notenoughupdates/miscgui/CalendarOverlay.java

```diff
@@ -1471,12 +1485,15 @@ public void drawTimer() {
 								tooltipToDisplay.add(sbEvent.display);
 								tooltipToDisplay.add(
 									EnumChatFormatting.GRAY + "Starts in: " + EnumChatFormatting.YELLOW + prettyTime(timeUntil, false));
+								addCountdownCalculatorToTooltip(timeUntil, tooltipToDisplay);
 								if (sbEvent.lastsFor >= 0) {
 									tooltipToDisplay.add(EnumChatFormatting.GRAY + "Lasts for: " + EnumChatFormatting.YELLOW +
 										prettyTime(sbEvent.lastsFor, true));
+									addCountdownCalculatorToTooltip(sbEvent.lastsFor + timeUntil, tooltipToDisplay);
 									if (timeUntil < 0) {
 										tooltipToDisplay.add(EnumChatFormatting.GRAY + "Time left: " + EnumChatFormatting.YELLOW +
 											prettyTime(sbEvent.lastsFor + timeUntil, true));
+										addCountdownCalculatorToTooltip(sbEvent.lastsFor + timeUntil, tooltipToDisplay);
```

Comment:
![image](https://github.com/NotEnoughUpdates/NotEnoughUpdates/assets/20768569/2a6d2eae-20a8-422a-b99f-1f3f8bd20e3b)
Maybe if we have a currently running event we can skip the first duplicate timestamp? Still giving this my approve because it works great already.